src/transport/tests/TestGroupMessageCounter.cpp

```diff
@@ -326,8 +326,11 @@ void ReorderPeerRemovalTest(nlTestSuite * inSuite, void * inContext)
     NL_TEST_ASSERT(inSuite, mGroupPeerMsgCounter.GetNodeIdAt(1, 0, false) == 9);
 }
 
+// Disabled for Zephyr since CHIP_CONFIG_MAX_FABRICS < 12
 #if !__ZEPHYR__
-
+#if CHIP_CONFIG_MAX_FABRICS < 12
```

Comment:
This is breaking the ESP32 CI.  Since this is not related directly to the counter tests, maybe do this as a separate PR?